pyproject.toml

```diff
@@ -34,6 +34,7 @@ tiktoken = {version = "^0.3.2", optional = true, python="^3.9"}
 pinecone-client = {version = "^2", optional = true}
 pinecone-text = {version = "^0.4.2", optional = true}
 weaviate-client = {version = "^3", optional = true}
+marqo = {version = "^0.9.1", optional=true}
```

Comment:
@hwchase17 anywhere else we need to add dependency?

Reply:
it was already added to `all` below, those are the two places